keyboards/technicpad/readme.md

```diff
@@ -0,0 +1,26 @@
+# TechnicPad
+
+![TechnicPad](https://i.imgur.com/LPD0KSL.jpg)
+
+A 4% (2x2) macropad, that is compatible to technic building blocks. 
+
+* Keyboard Maintainer: [Niclas Hirschfeld](https://github.com/nwhirschfeld)
+* Hardware Supported: Pro Micro boards
+* Hardware Availability: for now TechnicPad is handwired and uses an Arduino pro micro. [TechnicPad repository](https://github.com/nwhirschfeld/TechnicPad) 
```

Comment:
Should be relocated to `keyboards/handwired`

Reply:
https://github.com/nwhirschfeld/qmk_firmware/commit/b12a2cd4e00fd9e266def8c60ab2cae7dfaf2893